projects/developer/src/app/common/components/live-range-selector/component.scss

```diff
@@ -0,0 +1,77 @@
+.live-range-selector {
+    align-items: flex-end;
+
+    div {
+        margin-right: 20px;
+    }
+
+    .ui-inputgroup {
+        .ui-inputgroup-addon {
+            line-height:  normal;
+        }
+
+        ::ng-deep input,
+        ::ng-deep .ui-dropdown {
+            line-height:  normal;
+            border-top-left-radius: 0;
+            border-bottom-left-radius: 0;
+            flex: 1 1 auto;
+        }
+    }
+
+    .live-range-selector__date-filter-btn, .live-range-selector__date-filter-dropdown {
+        padding-top: 0;
+        padding-bottom: 0;
+    }
+
+    &.spaced {
+        .live-range-selector__date-filter-ranges, .live-range-selector__date-filter-dropdown {
+            margin-left: 55px;
+
+            button {
+                margin-left: 3px;
+            }
+        }
+    }
+
+    .live-range-icon {
+        margin-left: 5px;
+        margin-right: 5px;
+    }
+
+    .live-range-inactive {
+        color:grey
+    }
+
+    .live-range-active {
```

Comment:
Are these modifiers of the `.live-range-icon` ? Should they be `.live-range-selector__icon--active` etc. ?

Reply:
These were copy-pasted from the original temporal filter component. I'm not too worried about BEM in Angular components, since they already get scoped.